tools/pipelines/templates/build-npm-package.yml

```diff
@@ -425,6 +435,87 @@ stages:
               workingDir: ${{ parameters.buildDirectory }}
               customCommand: 'run bundle-analysis:run'
 
+          - ${{ if and(ne(variables['Build.Reason'], 'PullRequest'), ne(variables['System.TeamProject'], 'public')) }}:
+            - task: Bash@3
+              displayName: Create test directory for uploading telemetry
+              inputs:
+                targetType: 'inline'
+                script: |
+                  mkdir ${{ variables.testWorkspace }}
+
+            # Auth to internal feed
+            - task: Bash@3
+              name: Initialize
+              displayName: Initialize .npmrc for telemetry logger
+              inputs:
+                targetType: 'inline'
+                workingDirectory: ${{ variables.testWorkspace }}
```

Comment:
I was afraid of overriding it in case it exists - if you don't have a strong objection, I will keep it for now :)

Reply:
It shouldn't exist in the telemetry-generator folder since it doesn't exist in source control. I do lean towards making the change but wouldn't block the PR for it.

Reply:
Removed the new folder logic